src/data/grammar/verbs.yaml

```diff
@@ -96,9 +96,24 @@
       "doing pushups",
       "hibernating",
       "using the power of prayer",
-      "talking to a therapist",
       "doing hard drugs",
-      "lying face down and giving up"
+      "running head first into a wall",
+      "barfing into a hat",
+      "living inside an animal carcass",
+      "refusing to shake people's hands"
+  ],
+  "verbDesireableOutcomeIntransitive": [
+      "win the Pro Tour",
+      "secure a job with WOTC",
```

Comment:
maybe make a new list of companies, etc. WOTC, The DCI, ChannelFireball, Twitch.tv, Jim's Card Shack